src/index.ts

```diff
@@ -80,13 +81,17 @@ export class LinkChecker extends EventEmitter {
     let server: http.Server | undefined;
     const hasHttpPaths = options.path.find(x => x.startsWith('http'));
     if (!hasHttpPaths) {
-      const port = options.port || 5000 + Math.round(Math.random() * 1000);
+      let port = options.port;
       server = await startWebServer({
         root: options.serverRoot!,
         port,
         markdown: options.markdown,
         directoryListing: options.directoryListing,
       });
+      if (port === undefined) {
+        const addr = server.address() as AddressInfo;
```

Comment:
So does this mean the underlying web server is going to use it's own logic to grab a random port?